docs/CONFIG.md

```diff
@@ -38,6 +38,12 @@ Configuration lives in [config.yml](../config.yml).
   - `max_concurrent_access_control_syncs`: The maximum number of concurrent access control syncs. Defaults to 1.
   - `job_cleanup_interval`: The interval (in seconds) to run job cleanup task.
   - `log_level`: Connector service log level. Defaults to `INFO`.
+- `extraction_service`: Local extraction service-related configurations. These configurations are optional and are not included by default. The presence of these configurations enables local content extraction.
+  - `host`: The host of the local extraction service.
+  - `timeout`: Request timeout for local extraction service requests, in seconds. Defaults to 30.
+  - `use_file_pointers`: Whether or not to use file pointers for local extraction. Defaults to `false`.
+  - `stream_chunk_size`: The size that files are chunked to for streaming when sending a file to the local extraction service, in bytes. Only applicable if `use_file_pointers` is `false`. Defaults to 65536 (64KB).
+  - `shared_volume_dir`: The location for files to be extracted from. Only applicable if `use_file_pointers` is `true`. Defaults to `/app/files`.
```

Comment:
Would it make sense to add more info here, mentioning how to set up Tika to also use this directory?

Reply:
I think just a link is perfect :)